kubernetes-replicator.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: kubernetes-replicator
+  version: 2.10.2
+  epoch: 0
+  description: Kubernetes controller for synchronizing secrets & config maps across namespaces
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/mittwald/kubernetes-replicator
+      tag: v${{package.version}}
+      expected-commit: 8f4de965a57b974b446fae30048269894fd4ff3d
+
+  - uses: go/build
+    with:
+      packages: .
+      ldflags: |
+        -s -w
```

Comment:
```suggestion

```
`-s` is something we avoid and `-w` comes out of the box from `go/build` pipeline. ref: https://github.com/chainguard-dev/melange/blob/0900229dc8a4b90e8ad950ffb44e0acee502b996/pkg/build/pipelines/go/build.yaml#L58-L62

Reply:
looks like upstream don't set any version related ldflags in their goreleaser config and I was not able to find the other variable as well so we can omit the other ldflag as well.  

https://github.com/mittwald/kubernetes-replicator/blob/master/.goreleaser.yml

Reply:
Thank you. Now resolved.